integr8ly/tutorial-web-app
# remove steelthreads

Remove the old walkthrough directories and content.

Verification steps:

1. run the webapp in local dev mode
1. make sure all walkthroughs are still present
1. go through the walkthroughs and make sure they all still work
